brokerapi/brokers/storage/definition.go

```diff
@@ -151,8 +165,25 @@ func serviceDefinition() *broker.ServiceDefinition {
 				PlanId:          "c8538397-8f15-45e3-a229-8bb349c3a98f",
 				ProvisionParams: map[string]interface{}{"location": "us"},
 				BindParams: map[string]interface{}{
-					"role":     "storage.objectAdmin",
-					"location": "us-west1",
+					"role": "storage.objectAdmin",
+				},
+			},
+			{
+				Name:            "Regional Storage",
+				Description:     "Create a regional bucket with a service account that can create/read/delete the objects in it.",
```

Comment:
Missing the "list" permission, see: https://cloud.google.com/storage/docs/access-control/iam-roles